composer.json

```diff
@@ -9,22 +9,21 @@
 		"paragonie/halite": "^4.6|^5.0",
 		"doctrine/orm": "^2.5",
 		"doctrine/doctrine-bundle": "^2.0",
-		"symfony/property-access": "^4.1|^5.0|^6.0",
-		"symfony/dependency-injection": "^4.1|^5.0|^6.0",
-		"symfony/yaml": "^4.1|^5.0|^6.0",
-		"symfony/http-kernel": "^4.1|^5.0|^6.0",
-		"symfony/config": "^4.1|^5.0|^6.0",
+		"symfony/property-access": "^5.4|^6.0|^7.0",
+		"symfony/dependency-injection": "^5.4|^6.0|^7.0",
+		"symfony/yaml": "^5.4|^6.0|^7.0",
+		"symfony/http-kernel": "^5.4|^6.0|^7.0",
+		"symfony/config": "^5.4|^6.0|^7.0",
 		"doctrine/annotations": "^1.13|^2.0",
-		"symfony/expression-language": "^4.1|^5.0|^6.0"
+		"symfony/expression-language": "^5.4|^6.0|^7.0"
 	},
 	"require-dev": {
-		"phpunit/phpunit": "^8.0|^9.0",
 		"defuse/php-encryption": "^2.1",
 		"doctrine/cache": "^1.11",
 		"phpstan/phpstan": "^1.4",
 		"jetbrains/phpstorm-attributes": "^1.0",
 		"phpcompatibility/php-compatibility": "^9.3",
-		"symfony/phpunit-bridge": "^6.0"
+		"symfony/phpunit-bridge": "^6.4|^7.0"
```

Comment:
I think you can drop symfony/phpunit-bridge 6 completly. It is mostly backward compatible

Reply:
I checked the docs and you're right, we can drop the support for 6.4, especially since they are in the dev-requirements, which do not affect users of our library.  

From the docs:
> The PHPUnit bridge is designed to work with all maintained versions of Symfony components,
> even across different major versions of them.
> You should always use its very latest stable major version to get the most accurate deprecation report.